astropy/visualization/mpl_normalize.py

```diff
@@ -26,7 +26,7 @@
 )
 
 try:
-    import matplotlib  # noqa: F401
+    import matplotlib as mpl  # noqa: F401
```

Comment:
To be honest, I have no idea why this line is here! But it makes no sense to assign a different name if we don't even use the import at all.

Reply:
Might be accidentally left over from #3413 – has anyone ever tried to test without this?